extension/flink-doris-connector/build.sh

```diff
@@ -25,23 +25,45 @@
 
 set -eo pipefail
 
-ROOT=`dirname "$0"`
-ROOT=`cd "$ROOT"; pwd`
+usage() {
+  echo "
+  Eg.
+    $0 flink_version scala_version
```

Comment:
How about list all ?
```
sh build.sh 1.11.6 2.12
sh build.sh 1.12.7 2.12
sh build.sh 1.13.5 2.12
```

So that we don't need to find the document each time we try to build.
Same as Spark Connector